CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Enhancements
 * Added `MutableSubscriptionSet.removeByType` for removing subscriptions by their realm object type. (Issue [#317](https://github.com/realm/realm-dart/issues/317))
+* Support results of primitives, ie. `RealmResult<int>`. (Issue [#162](https://github.com/realm/realm-dart/issues/162))
```

Comment:
```suggestion
* Support `RealmResults` of primitive types (bool, int, string ...), ie. `RealmResults<int>`. (Issue [#162](https://github.com/realm/realm-dart/issues/162))
```

Reply:
I think this would be inconsistent. We have used the word primitives before in the CHANGELOG without defining it, and we often refer to the class `RealmList` as realm list(s) in free text.

Reply:
It sounds better to me. But it is just a suggestion. It is not important. As you decide.